frameit/frames_generator/Rakefile

```diff
@@ -79,4 +100,36 @@ def convert_image_resource(path)
   image.write(path)
 end
 
+def measure_slot(path)
+  tmp_output = Time.now.to_i.to_s + ".png"
+
+  convert = MiniMagick::Tool::Convert.new
+  convert << path
+  convert.negate
+  convert.define("connected-components:verbose=true")
+  convert.define("connected-components:area-threshold=100")
+  convert.connected_components("8")
+  convert.auto_level
+  convert << tmp_output
+  command_output = convert.call
+
+  sh("rm #{tmp_output}", verbose: false)
+
+  raw_dimensions = command_output.split("\n")[1].strip
+  dimensions = /(\d+)x(\d+)\+(\d+)\+(\d+)/.match(raw_dimensions)
+  x = dimensions[3]
+  y = dimensions[4]
+  width = dimensions[1]
+  height = dimensions[2]
+  return x.to_i, y.to_i, width.to_i, height.to_i
+end
+
+def sanitize_device_name(filename)
```

Comment:
This method is quite tricky, because we need to sanitize a device name based on the filename to be compliant with the current conventions , i.e. `Apple iPhone 7 Plus.png` => `iPhone 7 Plus`. If that bothers us, maybe it's worth to considering if we could put the full device name to `offsets.json` file? 🤔 Although don't have much context what we are doing with `offsets.json` next.

Reply:
Yeah, we don't want touch that part if it currently works ;)

Reply:
Fair enough. Let's leave it as it is then 👍